src/couch_db_updater.erl

```diff
@@ -294,6 +294,10 @@ handle_info({update_docs, Client, GroupedDocs, NonRepDocs, MergeConflicts,
             couch_event:notify(Db2#db.name, updated);
         true -> ok
         end,
+        case NonRepDocs2 of
```

Comment:
can we write this in the style of the previous notification? It's slightly shorter but, more importantly, it's consistent with the surrounding coding style.

if NonRepDocs2 /= [] ->
  couch_event:notify(...
true -> ok
end,